### PR TITLE
[CLOUD-2706] - update to use new logging-ext module from jboss-eap-modules

### PR DESCRIPTION
**Pull request by luck3y**
https://issues.jboss.org/browse/CLOUD-2706
Signed-off-by: Ken Wills <kwills@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[CLOUD-XYA] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

**Comment by luck3y** [comment]
EAP 6.4 logging change.

**Comment by luck3y** [comment]
@bstansberry ping for review :P

File: cp-overrides.yaml [hunk 1]
```diff
@@ -5,6 +5,10 @@ modules:
             git:
                   url: https://github.com/jboss-openshift/cct_module.git
                   ref: sprint-16
+          - name: jboss-eap-modules
+            git:
+                  url: https://github.com/jboss-container-images/jboss-eap-modules
+                  ref: sprint-16
```
Comment (bstansberry) [inline cp-overrides.yaml]: Did jboss.eap.logging.ext exist in sprint-16?

Comment (luck3y) [reply cp-overrides.yaml]: No .. I think that was why I didn't add them previously, will correct again, sorry. (Juggling some builds at the same time, I should have just waited ;( )

**Comment by luck3y** [comment]
I've reverted that change. We'll need to update them the next time we release the images, along with the merge from -dev.

**Review by bstansberry — APPROVED** [review]

**Pull request merged**